src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SchemaRepository.cs

```diff
@@ -1,4 +1,6 @@
-﻿using Microsoft.OpenApi;
+﻿using System.Diagnostics.CodeAnalysis;
+
+using Microsoft.OpenApi;
```

Comment:
Please revert this and just fully-qualify the attribute usage as it's the only thing that needs it.